smartsim/_core/control/controller.py

```diff
@@ -512,19 +598,26 @@ def _create_batch_job_step(
                 "EntityList must have batch settings to be launched as batch"
             )
 
+        telemetry_dir = telemetry_dir / entity_list.name
         batch_step = self._launcher.create_step(
             entity_list.name, entity_list.path, entity_list.batch_settings
         )
+        batch_step.meta["entity_type"] = str(type(entity_list).__name__).lower()
+        batch_step.meta["exp_path"] = self.exp_path
```

Comment:
I'm not sure that I like idea of `exp_path` being an attr on the `Controller` that is only ever set to a "proper" value in `_launch`. Not that would ever expect someone to do something like
```py
Controller()._create_job_step(model, "path/to/a/dir")
```
but it _could_ be done.

I also can't find a good way to remove the attr either tho that doesn't just devolve into just param diving. If you can find a good way to remove it that would be much appreciated, but otherwise don't sweat it and we can add this onto the list of things to address post this release.

Reply:
Removed the unused `step.meta['exp_path']` and removed the associated member variable on the controller. Per discussion, a refactor of the manifest builder may be necessary to avoid passing `exp_path` into the launch/start methods of controller